test/decoration_tests/decoration_tests.jl

```diff
@@ -35,20 +35,20 @@ let b
     # @test_throws ArgumentError @decorated(BigInt(1), 1//10)
 
     # Tests related to powers of decorated Intervals
-    @test @decorated(2,3) ^ 2 == DecoratedInterval(4, 9)
-    @test @decorated(2,3) ^ -2 == DecoratedInterval(1/9,1/4)
-    @test @decorated(-3,2) ^ 3 == DecoratedInterval(-27., 8.)
-    @test @decorated(-3,-2) ^ -3 == DecoratedInterval(-1/8.,-1/27)
-    @test @decorated(0,3) ^ 2 == DecoratedInterval(0, 9)
-    @test @decorated(0,3) ^ -2 == DecoratedInterval(1/9, Inf, trv)
-    @test @decorated(2,3)^Interval(0.0, 1.0) == DecoratedInterval(1.0,3.0)
-    @test @decorated(2,3)^@decorated(0.0, 1.0) == DecoratedInterval(1.0,3.0)
-    @test @decorated(0, 2)^Interval(0.0, 1.0) == DecoratedInterval(0.0,2.0, trv)
-    @test @decorated(0, 2)^@decorated(0.0, 1.0) == DecoratedInterval(0.0,2.0, trv)
-    @test @decorated(-3, 2)^Interval(0.0, 1.0) == DecoratedInterval(0.0,2.0, trv)
-    @test @decorated(-3, 2)^@decorated(0.0, 1.0) == DecoratedInterval(0.0,2.0, trv)
-    @test @decorated(-3, 2)^Interval(-1.0, 1.0) == DecoratedInterval(0.0,Inf, trv)
-    @test @decorated(-3, 2)^@decorated(-1.0, 1.0) == DecoratedInterval(0.0, Inf, trv)
+    @test @decorated(2,3) ^ 2 ≛ DecoratedInterval(4, 9)
```

Comment:
the ≛ is the standard compliant right? If it is, then it doesn't check that the decoration matches, should use either === or check for decoration separately (again, ITF1788.jl takes care of this, so it hopefully will help minimize manual writing)